master/buildbot/worker/docker.py

```diff
@@ -153,7 +153,7 @@ def checkConfig(self, name, password, docker_host, image=None,
                 if not isinstance(volume_string, str):
                     continue
                 try:
-                    bind, volume = volume_string.split(":", 1)
+                    _, __ = volume_string.split(":", 1)
```

Comment:
Perhaps add a comment above like "Note that here we rely on tuple unpacking raising ValueError if the number of elements is wrong".